pkg/hintrunner/zero/zerohint_usort.go

```diff
@@ -26,3 +28,66 @@ func newUsortEnterScopeHint() hinter.Hinter {
 func createUsortEnterScopeHinter() (hinter.Hinter, error) {
 	return newUsortEnterScopeHint(), nil
 }
+
+func newUsortVerifyMultiplicityBodyHint(nextItemIndex hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "UsortVerifyMultiplicityBody",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> current_pos = positions.pop()
+			//> ids.next_item_index = current_pos - last_pos
+			//> last_pos = current_pos + 1
+
+			positionsInterface, err := ctx.ScopeManager.GetVariableValue("positions")
+			if err != nil {
+				return err
+			}
+
+			positions, ok := positionsInterface.([]uint64)
+			if !ok {
+				return err
+			}
+
+			current_pos, err := utils.Pop(&positions)
+			if err != nil {
+				return err
+			}
+
+			last_pos, err := ctx.ScopeManager.GetVariableValue("last_pos")
+			if err != nil {
+				return err
+			}
+
+			// Calculate `next_item_index` memory value
+			newNextItemIndexValue := current_pos - last_pos
+			newNextItemIndexMemoryValue := memory.MemoryValueFromUint(newNextItemIndexValue)
+
+			// Save `next_item_index` value in address
+			addrNextItemIndex, err := nextItemIndex.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			err = vm.Memory.WriteToAddress(&addrNextItemIndex, &newNextItemIndexMemoryValue)
+			if err != nil {
+				return err
+			}
+
+			// Save `current_pos` and `last_pos` values in scope variables
+			err = ctx.ScopeManager.AssignVariables(map[string]any{
```

Comment:
Sorry, in previous comment I pointed that you can just return err. Actually you can just return what this function returns haha 😅